python-package/securexgboost/core.py

```diff
@@ -1103,7 +1102,7 @@ def attest(self, verify=True):
         ----------
         verify: bool
             Whether to verify the enclave report or not
-            Warning: Should only be set to False for development and testing.
+            .. note:: ``verify`` should be set to ``False`` only for development and testing in simulation mode
```

Comment:
Remove the `.. note::` here? Do we need the double backticks around `verify` and `False`? Will they make a difference when the documentation is generated? i.e. can we go without any backticks?

Reply:
Without backticks, the words appear like regular text instead of keywords.
Chaning ".. note" to ".. warning" so it pops up in a box.